### ISSUE TITLE
sidewalks?

### ISSUE THREAD

**Issue by alibama** [pietervdvn/MapComplete#210]
I would like to help make sidewalks a mapcomplete map so that people in wheelchairs can use openstreetmap for navigation

**Comment by pietervdvn** [comment]
Hey,

Really cool and thanks for reaching out. Currently I do not have the time to implement your theme request, but if you have some technical knowledge (aka not afraid of a JSON) you can make this theme yourself. Please see the documentation here: https://github.com/pietervdvn/MapComplete/blob/master/Docs/Making_Your_Own_Theme.md

**Comment by alibama** [comment]
This is great news - I'm going to put this process in front of some colleagues. Thank you so much!  This is a game changer for getting beginners started.

**Comment by pietervdvn** [comment]
Yeah, it is made to get beginners started.

Be very careful when creating the preset though, the wording etc. can be the difference between faulty data etc.

Furthermore, I'd love to implement an extension that would present streets as two streets - a left segment and right segment to be able to have a popup for both sides [just like this one](https://zlant.github.io/parking-lanes/#18/52.48275/13.43426) - but I do lack the (free) time for that right now.

**Comment by pietervdvn** [comment]
BTW, my day-to-day job is creating routeplanners, within the small company [Anyways](anyways.eu). If you are interested, we can create a routeplanner for wheelchair users; however this is quite an undertaking and the data is still largely missing right away.

**Comment by alibama** [comment]
Hi Pieter - just checked out Anyways - very cool!  I think that's probably our dream, but i know we need better data before we start that conversation.  

by the way - i watched https://www.youtube.com/watch?v=nVbFrNVPxPw and it says i need 500 edits?  is that a hard requirement?  I do most of my updates in bulk through JOSM - that's going to take me a year to use?

**Comment by pietervdvn** [comment]
Hi Alibama,

You need 500 edits to be able to use the CustomThemeGenerator-page. This is to prevent new people of discovering that page and fucking things up - e.g. they misunderstood the parking tag and add parking spots where the parked their camper...

You do NOT need 500 edits to create a custom theme, it is just a bit harder to do. Follow the docs mentioned earlier, make a .json, base64-encode it and load it as the hash in the URL and you are good to go.

I wanted to have a certain threshold for making a custom theme, as a custom theme could cause a lot of harm and to protect myself for a lot of newbies asking newbie questions.

Alternatively, get in touch with your local OSM community and ask help from a seasoned mapper. Otherwise, I can also create the theme for you, but as my spare time is kinda gone, that won't be as volunteer but as a freelancer (aka: your gonna have to buy my time)

**Comment by matkoniecz** [comment]
> BTW, my day-to-day job is creating routeplanners, within the small company Anyways. 

FYI, this link is broken. https://www.anyways.eu/ seems to be the indented one

**Comment by pietervdvn** [comment]
Indeed, edited

**Comment by pietervdvn** [comment]
Hi @alibama 

Did you make any progress on your theme by now?

**Comment by alibama** [comment]
Hey!  actually we've been working in the task manager, https://tasks.openstreetmap.us/projects/239 and that let us set up which tools were available inside of the space

we also are using https://tasks.opensidewalks.com/projects/9 and one of the neat things that got done there is that the map got broken down in to single intersection spots which was really a big UI improvement... let me circle back around here with some of our new volunteers and see if we can get a release made...

**Comment by pietervdvn** [comment]
[There is a sidewalk-centric theme now](https://pietervdvn.github.io/mc/develop/sidewalks.html?z=18&lat=51.21189&lon=3.217454&language=en#welcome) which uses left:right on a main highway.
